Redirect governance documents from wiki to jenkins.io

MarkEWaite opened the pull request:
## Redirect Governance Documents from wiki to jenkins.io

The governance document and governance meeting pages have already migrated and are only pointers to the official jenkins.io pages.

The governance board page has been migrated to jenkins.io but does not have a pointer in the page.  This redirect avoids editing that page by redirecting to jenkins.io.

The candidacy process and election process for the Jenkins board are maintained on jenkins.io.  No need to show the user an outdated copy from the wiki.

Would like review from @oleg-nenashev, @slide, and other members of the Jenkins board to assure that I've not missed something in proposing this redirect.

MarkEWaite committed:
Redirect governance documents from wiki to jenkins.io

The governance document and governance meeting pages have already migrated
and are only pointers to the official jenkins.io pages.

The governance board page has been migrated to jenkins.io but does not
have a pointer in the page.  This redirect avoids editing that page by
redirecting to jenkins.io.

The candidacy process and election process for the Jenkins board are
maintained on jenkins.io.  No need to show the user an outdated copy
from the wiki.

oleg-nenashev commented on dist/profile/templates/confluence/vhost.conf:
```diff
@@ -2977,6 +2977,11 @@ RewriteRule "^/display/JENKINS/Logging$" "https://jenkins.io/doc/book/system-adm
+RewriteRule "^/display/JENKINS/Board\+Candidacy\+Process" "https://jenkins.io/project/board-election-process/" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Board\+Election\+Process$" "https://jenkins.io/project/board-election-process/" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Governance\+Board$" "https://jenkins.io/project/board/" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Governance\+Document$" "https://jenkins.io/project/governance/#compatibility-matters" [NC,L,QSA,R=301]
```
```suggestion
RewriteRule "^/display/JENKINS/Governance\+Document$" "https://jenkins.io/project/governance/" [NC,L,QSA,R=301]
```

oleg-nenashev reviewed (changes requested):
1 minor issue with anchor

MarkEWaite committed:
Redirect to full governance, not a subset

The compatibility matters section is a close
match to the original document but the new
document is best viewed in its entirety rather
than jumping into a specific section.

Co-Authored-By: Oleg Nenashev <o.v.nenashev@gmail.com>

olblak committed:
Merge branch 'staging' into redirect-governance-docs-from-wiki-to-jenkins-io

olblak committed:
Merge branch 'staging' into redirect-governance-docs-from-wiki-to-jenkins-io
